playbooks/pas_gcp_dev.yml

```diff
@@ -0,0 +1,20 @@
+---
+- hosts: pas_gcp_dev
```

Comment:
Here too - a good name for this one might be `- name: build a test instance of the Princeton and Slavery site on gcloud`.